explorer/interpreter/interpreter.cpp

```diff
@@ -182,7 +184,15 @@ class Interpreter {
       -> ErrorOr<Success>;
 
   auto CallDestructor(Nonnull<const DestructorDeclaration*> fun,
-                      Nonnull<const Value*> receiver) -> ErrorOr<Success>;
+                      Nonnull<const Value*> receiver,
+                      const LocationValue* class_addr) -> ErrorOr<Success>;
+
+  // If the given `method` has a self argument, bind it to `method_scope`.
+  void BindSelfIfPresent(Nonnull<const CallableDeclaration*> decl,
+                         Nonnull<const Value*> receiver,
+                         Nonnull<const Value*> receiver_addr,
+                         const SourceLocation& source_location,
```

Comment:
By convention, `SourceLocation` parameters tend to go last.

Reply:
Moved, thanks.